### PR TITLE
[S] Update README, LICENSE & CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by Istar-Eldritch**
What the title says.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,124 @@
...
+
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://choosealicense.com/licenses/lgpl-3.0/)  
+[![npm version](https://badge.fury.io/js/%40repositive%2Fquery-parser.svg)](https://badge.fury.io/js/%40repositive%2Fquery-parser)  
+[![CircleCI](https://circleci.com/gh/repositive/query-parser.svg?style=svg)](https://circleci.com/gh/repositive/query-parser)  
+
+* [Installation](#installation)
+* [Available Serializers](#available-serializers)
+* [Usage](#usage)
+
+The purpose of this library is to transform search string to a intermediary tokenized data structure that can be used to perform analysis on the search or middle ground to translate the query into other DSL or query structures.
```
Comment (DennisSchwartz) [inline README.md]: The purpose of this library is to transform a search string to a ~~intermediary~~ tokenized data structure that can be used to perform analysis on the search input or as an intermediate structure to translate the query into other DSLs or query structures.

**Comment by Istar-Eldritch** [comment]
Addressed comments. Thank you for helping with the wording on that crappy description :smile:

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 * [Available Serializers](#available-serializers)
 * [Usage](#usage)
 
-The purpose of this library is to transform search string to a intermediary tokenized data structure that can be used to perform analysis on the search or middle ground to translate the query into other DSL or query structures.
+The purpose of this library is to transform a search string to a intermediary tokenized data structure that can be used to perform analysis on the search input or as an intermediate structure to translate the query into other DSLs or query structures.
```
Comment (DennisSchwartz) [inline README.md]: Sorry, there is one more little thing...
It's `a**n** intermediary tokenized`. Sorry if I missed this before.

Comment (DennisSchwartz) [inline README.md]: Ah wait!

Comment (DennisSchwartz) [inline README.md]: My previous request was aimed to remove the first intermediary altogether. Because we use intermediate later in the sentence.

**Comment by Istar-Eldritch** [comment]
Done (hopefully)

**Pull request merged**